react/features/security/components/security-dialog/web/PasswordSection.tsx

```diff
@@ -132,7 +121,7 @@ function PasswordSection({
         if (notifyMode === NOTIFY_CLICK_MODE.ONLY_NOTIFY) {
```

Comment:
`if (!notifyMode || notifyMode === NOTIFY_CLICK_MODE.ONLY_NOTIFY) {` ?

Reply:
Yes. Decided to put this in another PR.